### PR TITLE
Docs Party 2024

### PR DESCRIPTION
**Pull request by notpeter**
This is a `docs_party` branch with a number of documentation improvements.

See 
- [`#docs`](https://zed.dev/channel/docs-8794/notes) (Zed Channel)
- [`#docs`](https://zed-industries.slack.com/archives/C04TX5CDT63) (Slack Channel)

A rough `To Be Documented` list can be found by searching for `TBD`:

```shell
git fetch && git checkout docs_party && git pull

# See all TBDs:
grep -R '^TBD:' docs

# See 5 random TBDs:
grep -R '^TBD:' docs | shuf -n 5
```

There are additional things that need documentation improvements in the Zed `#docs` channel.

Release Notes:

- Docs Party 2024

### CODE REVIEW

**Comment by myromeu** [comment]
@notpeter hi. Can I submit little (but so annoying) fix for outdated keybinding documention to this branch? It's not worth it to open new issue and do another PR because one word fix)

**Comment by bajrangCoder** [comment]
@\notpeter I have done some improvement in language.md , but I'm unable to push here.

Sorry for any inconveniences 

Check [this](https://github.com/bajrangCoder/zed/tree/docs_party)

**Comment by notpeter** [comment]
> @\notpeter I have done some improvement in language.md , but I'm unable to push here.

I cherry-picked your changes in d4ae81b0ca100040ecc8abfa057e535ffd1185cf. Thanks for this!

**Comment by notpeter** [comment]
> @notpeter hi. Can I submit little (but so annoying) fix for outdated keybinding documention to this branch? It's not worth it to open new issue and do another PR because one word fix)

Commit it to a branch on a fork and I will happily cherry-pick.

**Comment by myromeu** [comment]
@notpeter completed there: https://github.com/myromeu/zed/tree/docs_party

**Comment by notpeter** [comment]
> @notpeter completed there: https://github.com/myromeu/zed/tree/docs_party

@myromeu 
1. Can you ensure the email in your commit is associated with your GitHub account?
2. Can you sign our [Contributor Licensing Agreement](https://zed.dev/cla) (scroll down to the bottom and Sign in with Github)

**Review by notpeter — COMMENTED** [review]
I've review all 99 changed files and I think they look good on content.

Fixing links with [mdbook-linkcheck](https://crates.io/crates/mdbook-linkcheck) now.

File: docs/src/repl.md [hunk 1]
```diff
@@ -73,52 +76,73 @@ On MacOS, your system Python will _not_ work. Either set up [pyenv](https://gith
...
+-->
+
+### Scala
+
+- Install Scala with `cs setup` (Coursier): https://www.scala-lang.org/download/
+  - `brew install coursier/formulas/coursier && cs setup`
+- REPL (Almond) Setup Instructions https://almond.sh/docs/quick-start-install
+  - `brew install --cask temurin` (Eclipse foundation official OpenJDK binaries)
+  - `brew install coursier/formulas/coursier && cs setup`
+  - `coursier launch --use-bootstrap almond -- --install`
```
Comment (rgbkrk) [inline docs/src/repl.md:145]: Rad that you added `coursier` and almond instructions here.

**Pull request merged**